app/models/document.rb

```diff
@@ -13,7 +13,7 @@ class Document < ApplicationRecord
   has_many :editions, inverse_of: :document
   has_many :edition_relations, dependent: :destroy, inverse_of: :document
 
-  has_one  :published_edition,
+  has_one  :live_edition,
```

Comment:
If we decide we want to rename this (i think we should), then this work can be pulled out and completed in a separate PR, prior to adding the new columns to the documents table.

Reply:
I'm going to pull this out into a separate PR, as you said. It seems like a reasonable change.

Annoyingly I can't just cherry-pick this commit into a new branch because it follows all the other changes 😢 

So just wrapping my head around it so I can figure out how to re-create it from `main`.